ui/lib/core/addon/templates/components/confirm.hbs

```diff
@@ -0,0 +1,16 @@
+{{! template-lint-disable no-inline-styles}}
```

Comment:
Looks like this snuck back in 🙈

Reply:
ah yes, that's right. it turns out we do need the rule disabled here because of the `style={{style}}` below ☹️